content/en/tracing/faq/php-tracer-manual-installation.md

```diff
@@ -15,10 +15,63 @@ From the location you downloaded the package:
 
 ```bash
 $ tar -xf datadog-php-tracer.tar.gz -C /
```

Comment:
```suggestion
$ tar -xf datadog-php-tracer.tar.gz -C
```

Reply:
@SammyK the final `/` is necessary if we keep using `-C` ([docs](https://linux.die.net/man/1/tar)). did you mean also to replace `-C` with some other flag?

Reply:
Oh no, I didn't read the command correctly. 😞 Please ignore this suggestion. Sorry for the noise!

Reply:
ahahah thanks Sammy, sounds good.